artemis/config.py

```diff
@@ -246,6 +246,31 @@ class Nuclei:
                 "when the Github API rate limits are spent).",
             ] = get_config("NUCLEI_CHECK_TEMPLATE_LIST", default=True, cast=bool)
 
+            NUCLEI_TEMPLATE_GROUPS: Annotated[
+                List[str],
+                "A list (in the form of template1=group1,template2=group2,...) of template group assignments. If a "
+                "template is assigned to a group, instead of the template, the whole group will be reported as the "
+                "detected template name. Therefore, due to findings deduplication, only one instance of such "
+                "vulnerability will be reported. This is useful to detect situations when multiple .env detectors "
+                "detect a single file or multiple XSS templates are triggered on a single page.",
+            ] = get_config(
+                "NUCLEI_TEMPLATE_GROUPS",
+                default=",".join(
```

Comment:
it can be stored in json tbh, a lot simpler and more suitable for this structure